e2e/regression_test/settings.no_basal_rates.spec.js

```diff
@@ -0,0 +1,24 @@
+const { setup, match, loopSettings, FilterSettings, SettingDefault, SettingType } = require('../../src/index');
+
+describe('Closed loop is not allowed when settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+    });
+    it('are not applied for basal rates', async () => {
+        let config = {
+            scenario: 'flat_cgm_trace',
+            settings: FilterSettings(SettingDefault, [SettingType.BasalRates])
+        };
+        await loopSettings.Configure(config);
```

Comment:
[nit] Per our discussion last week, you can get rid of all of the extra `await`s.